x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_clone/clone_action_name.tsx

```diff
@@ -416,13 +424,42 @@ export const useNavigateToWizardWithClonedJob = () => {
       if (dv !== undefined) {
         sourceIndexId = dv.id;
       } else {
-        toasts.addDanger(
-          i18n.translate('xpack.ml.dataframe.analyticsList.noSourceDataViewForClone', {
-            defaultMessage:
-              'Unable to clone the analytics job. No data view exists for index {dataView}.',
-            values: { dataView: sourceIndex },
-          })
-        );
+        toasts.addDanger({
+          title: toMountPoint(
+            wrapWithTheme(
+              <>
+                <FormattedMessage
+                  id="xpack.ml.dataframe.analyticsList.noSourceDataViewForClone"
+                  defaultMessage="Unable to clone the analytics job. No data view exists for index {dataView}."
+                  values={{ dataView: sourceIndex }}
+                />
+                {canCreateDataView ? (
+                  <EuiText size="xs" color="text">
+                    <FormattedMessage
+                      id="xpack.ml.dataframe.analytics.cloneAction.dataViewPromptLink"
+                      defaultMessage="{linkToDataViewManagement}"
+                      values={{
+                        linkToDataViewManagement: (
+                          <EuiLink
+                            href={`${basePath.get()}/app/management/kibana/dataViews/create`}
+                            target="_blank"
+                          >
+                            <FormattedMessage
+                              id="xpack.ml.dataframe.analytics.cloneAction.dataViewPromptLinkText"
+                              defaultMessage="Create a data view for {dataView}"
+                              values={{ dataView: sourceIndex }}
```

Comment:
Super duper nit: Similarly, the variable name `dataView` in this message is a bit confusing as the data view has not been created yet. Suggestion to rename variable to something like `Create a data view for {sourceIndex}`.

Reply:
Updated in 7e8031b7f25662c8389e409f70fc8b762ff6fd8e